internal/pkg/cli/storage_init.go

```diff
@@ -582,6 +713,8 @@ are injected into your containers as environment variables for easy access.`,
 	cmd.Flags().BoolVar(&vars.noLSI, storageNoLSIFlag, false, storageNoLSIFlagDescription)
 	cmd.Flags().BoolVar(&vars.noSort, storageNoSortFlag, false, storageNoSortFlagDescription)
 
+	cmd.Flags().StringVar(&vars.parameterGroup, "param", "", "paramee")
+
```

Comment:
There's a nifty `.MarkHidden` function if you want to make the flag then hide it.
Ex:
```
	cmd.Flags().StringVarP(&vars.githubAccessToken, githubAccessTokenFlag, githubAccessTokenFlagShort, "", githubAccessTokenFlagDescription)
	_ = cmd.Flags().MarkHidden(githubAccessTokenFlag)
```

Reply:
https://pkg.go.dev/github.com/spf13/pflag#readme-hidden-flags

Reply:
Yayyy thank you for sharing the doc!! 

I will remove this line (I forgot to do this), and add the flags in a separate PR along with docs, so that all description-related modifications can be reviewed altogether in one PR. (At least that's my plan)

I will make use of `.MarkHidden` in the future!!